src/components/ErrorHandling/types.ts

```diff
@@ -0,0 +1,3 @@
+export type CustomBugsnagError = Error & {
```

Comment:
Should we rename this since it also appears to be used for Sentry?

Reply:
https://test-org-2v.sentry.io/discover/spruce:d629348361af4255bdbdccaa06a26410/?field=title&field=event.type&field=project&field=user.display&field=timestamp&field=replayId&homepage=true&name=All+Events&project=4505211042856960&query=&sort=-timestamp&statsPeriod=14d&yAxis=count()#breadcrumbs
The error sent here is not being picked up correctly by sentry and doesn't include the additional metadata. We may need to use a different format.
<img width="922" alt="image" src="https://github.com/evergreen-ci/spruce/assets/4605522/527a9773-50a5-42f4-aa9f-a74f8b9394cc">

Reply:
Good catch, here's what metadata looks like now https://test-org-2v.sentry.io/issues/4271518447/?query=is%3Aunresolved&referrer=issue-stream&stream_index=0#context-metadata